### PR TITLE
implement design for careers page

### PR DESCRIPTION
**Pull request by erizocosmico**
careers
careers_lightbox

(the glitch on the lightbox is because of the extension i used to make the screenshot)

### CODE REVIEW

File: src/sass/pages/career.scss [hunk 1]
```diff
@@ -2,4 +2,208 @@
...
+                    background-color: $color-careers-benefits-dot;
+                    height: $size-careers-benefits-list-item-dot;
+                    width: $size-careers-benefits-list-item-dot;
+                    border-radius: 50%;
+                }
+            }
+        }
+    }
+
+    .hq-gallery {
```
Comment (dpordomingo) [inline src/sass/pages/career.scss]: since the `hq-gallery` block has no custom style, I'd lower 1 indentation level doing:
```.hq-gallery__item {```

**Comment by marnovo** [comment]
Just change the `our headquarters` heading to `our offices`.
Is the "broken" lightbox image something we should be aware of?

**Comment by erizocosmico** [comment]
Nope, it's from the thing I used to take the screenshot:

> (the glitch on the lightbox is because of the extension i used to make the screenshot)

**Comment by erizocosmico** [comment]
Done @marnovo

**Pull request merged**